Trimble.Modus.Components/Controls/BorderlessEditor/BorderlessEditor.cs

```diff
@@ -3,6 +3,7 @@ namespace Trimble.Modus.Components;
 internal class BorderlessEditor : Editor
 {
     public BorderlessEditor()
-    {
+    {
+        BackgroundColor = Colors.Transparent;

```

Comment:
Is this also necessary?

Reply:
Yes we need this for https://github.com/trimble-oss/modus-mobile-maui-components/issues/419.
In groundworks dark mode we are seeing two views in TMInput, to fix this i have added this line
![image](https://github.com/user-attachments/assets/1a6fbf66-1718-46d6-a6e8-27e1404632bc)